iampolicymanagementv1/iam_policy_management_v1_integration_test.go

```diff
@@ -235,6 +235,32 @@ var _ = Describe("IAM Policy Management - Integration Tests", func() {
 			Expect(policy.Roles[0].RoleID).To(Equal(options.Roles[0].RoleID))
 			Expect(policy.Resources).To(Equal(options.Resources))
 
+			testPolicyETag = detailedResponse.GetHeaders().Get(etagHeader)
+
+		})
+	})
+
+	Describe("Patch an access policy", func() {
+
+		It("Successfully patched an access policy", func() {
+			shouldSkipTest()
+			Expect(testPolicyId).To(Not(BeNil()))
+			Expect(testPolicyETag).To(Not(BeNil()))
+
+			// Construct an instance of the PatchPolicyOptions model
+			options := new(iampolicymanagementv1.PatchPolicyOptions)
+			options.PolicyID = core.StringPtr(testPolicyId)
```

Comment:
This is fine the way you have it, but the core XXXPtr() functions are really only needed when dealing with literal values.  For example, in Go you can't use something like &"some string" to take the address of the literal string, hence the need for the XXXPtr() functions.
```suggestion
			options.PolicyID = &testPolicyId
```